app/lib/submission_builder/ty2024/states/md/documents/md502_su.rb

```diff
@@ -11,6 +11,12 @@ class Md502Su < SubmissionBuilder::Document
             def document
               build_xml_doc("Form502SU", documentId: "Form502SU") do |xml|
                 xml.Subtractions do |subtractions|
+                  if calculated_fields.fetch(:MD502_SU_LINE_AB).positive?
+                    subtractions.OtherDetail do |other_detail|
+                      other_detail.Code "ab"
```

Comment:
Needs to be captial "AB" - it's a enum

Reply:
There is probably a validation error, we should be catching those in tests :(

Reply:
Fixed!

Reply:
I will add both!

Reply:
oof sorry I missed this in code review